### PR TITLE
order_item_fact changed to dim_order_item as this is SCD1 load type

### PR DESCRIPTION
**Pull request by madhu-jain**

### CODE REVIEW

File: definitions/sources/declare_bronze.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 
-const table = ["stg_products","stg_order_items","stg_orders"]
+const table = ["stg_products","stg_order_items","stg_orders","stg_inventory_items"]
```
Comment (warunk9) [inline definitions/sources/declare_bronze.js:3]: can we put this on sqlx or else we need this file to be present

File: definitions/transformation/dim_order_items.sqlx [hunk 2]
```diff
@@ -0,0 +1,28 @@
+config { 
+    type: "incremental",
+    uniqueKey: ["id"],
```
Comment (warunk9) [inline definitions/transformation/dim_order_items.sqlx:3]: this file u have added , is it not present earlier?

**Review by warunk9 — COMMENTED** [review]
check comments , rest looks good

File: definitions/sources/declare_bronze.js (hunk 1 above)
Comment (madhu-jain) [reply definitions/sources/declare_bronze.js:3]: you want this sqlx to show append logic using operations?

Comment (warunk9) [reply definitions/sources/declare_bronze.js:3]: no no , just the declare part only

**Review by warunk9 — APPROVED** [review]

**Pull request merged**